wfl/fit/mace.py

```diff
@@ -0,0 +1,175 @@
+import os, sys, yaml, subprocess
+import warnings
+import ase.io
+from expyre import ExPyRe
+from wfl.configset import ConfigSet, OutputSpec
+from wfl.autoparallelize.utils import get_remote_info
+from expyre.resources import Resources
+from pathlib import Path 
+from shutil import copyfile
+
+
+def fit(fitting_configs, mace_name, mace_fit_params, mace_fit_cmd, ref_property_prefix="REF_", 
+        skip_if_present=True, run_dir=".", verbose=True, dry_run=False, remote_info=None,
+        remote_label=None, wait_for_results=True, prev_checkpoint_file=None, fit_cfgs_fname=None):
+    """
+        Fit MACE model.
+
+    Parameters
+    ----------
+    fitting_configs: ConfigSet
+       set of configurations to fit 
+    mace_name: str
+        name of MACE label
+    mace_fit_params: dict
+        parameters for fitting a MACE model.
+    mace_fit_cmd: str
+        command for excecuting the MACE fitting. (For example, "python ~/path_to_mace_cripts/run_train.py")
+    ref_property_prefix: str, default "REF\_"
+        string prefix added to atoms.info/arrays keys (energy, forces, virial, stress)
+    skip_if_present: bool, default False
+        skip if final MACE file exists in expected place
+    run_dir: str, default '.'
+        directory to run fitting in
+    verbose: bool default True
+        verbose output
+    dry_run: bool, default False
+        do a dry run, and returns fitting command including keywords
+    remote_info: dict or wfl.autoparallelize.utils.RemoteInfo, or '_IGNORE' or None
+        If present and not None and not '_IGNORE', RemoteInfo or dict with kwargs for RemoteInfo
+        constructor which triggers running job in separately queued job on remote machine.  If None,
+        will try to use env var WFL_EXPYRE_INFO used (see below). '_IGNORE' is for
+        internal use, to ensure that remotely running job does not itself attempt to spawn another
+        remotely running job.
+    remote_label: str, default None
+        label to match in WFL_EXPYRE_INFO
+    wait_for_results: bool, default True
+        wait for results of remotely executed job, otherwise return after starting job
+    prev_checkpoint_file: str, default None
+        Previous checkpoint file to restart from. 
+    fit_cfgs_fname: str, default None
```

Comment:
As I said elsewhere, I don't want to special case an xyz file for the fitting configs, since you can always pass a `ConfigSet` which wraps one file. The nice thing about doing it the wfl way (i.e. a ConfigSet) is that the contents will be automatically sent to a remote machine if needed, unlike a special file argument where that'd have to be dealt with manually.